packages/sdk/package.json

```diff
@@ -20,13 +20,15 @@
     "main": "dist/index.js"
   },
   "dependencies": {
+    "crypto-js": "^4.2.0",
```

Comment:
This dep [weighs in at 487kB](https://www.npmjs.com/package/crypto-js). Not sure if you looked at any built in support? There seems to be some [built in support for browsers](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#converting_a_digest_to_a_hex_string) given that the app runs on https. We should probably look into this or lightweight alternatives.